Offseason2022/src/main/java/frc/robot/commands/IntakeDeploy.java

```diff
@@ -23,7 +26,12 @@ public IntakeDeploy(boolean position)
   // Called when the command is initially scheduled.
   @Override
   public void initialize( )
-  {}
+  {
+    DataLogManager.log(getSubsystem( ) + ((m_intakeExtend) ? "DEPLOY" : "STOW"));

```

Comment:
We can remove the DataLogManager.log() calls from all Commands (can tell this from "IntakeDeploy extends CommandBase on line 13). These calls are done now for all Commands in RobotInit--not in every command.